tools/maxquant/macros.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <macros>
-    <token name="@VERSION@">1.6.10.43</token>
+    <token name="@VERSION@">1.6.10.44</token>
```

Comment:
Oh, no this will not work, there is no conda package for this version.

Please do it this way: https://github.com/galaxyproject/tools-iuc/blob/4a8f640dec78899be470ddd1c436fc0d158c80f3/tools/snap/snap_training.xml#L2

We do not increase the conda version, but the Galaxy tool version, as we added new parameters.